From c994d9343b17b269c1fe48eb26c4be7cbf991478 Mon Sep 17 00:00:00 2001
From: Jens Axboe <axboe@kernel.dk>
Date: Mon, 23 Dec 2019 21:02:57 -0700
Subject: [PATCH] test/accept-reuse: fix C99'ism

Signed-off-by: Jens Axboe <axboe@kernel.dk>
---
 test/accept-reuse.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/accept-reuse.c b/test/accept-reuse.c
index 955305b..5f3e19a 100644
--- a/test/accept-reuse.c
+++ b/test/accept-reuse.c
@@ -41,7 +41,7 @@ int main(int argc, char **argv)
 	struct addrinfo hints;
 	struct sockaddr sa;
 	socklen_t sa_size = sizeof(sa);
-	int ret, listen_fd, connect_fd, val;
+	int ret, listen_fd, connect_fd, val, i;
 
 	memset(&params, 0, sizeof(params));
 	ret = io_uring_queue_init_params(1024, &io_uring, &params);
@@ -130,7 +130,7 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
-	for (int i = 0; i < 2; i++) {
+	for (i = 0; i < 2; i++) {
 		struct io_uring_cqe *cqe = NULL;
 
 		ret = io_uring_wait_cqe(&io_uring, &cqe);
-- 
1.8.3.1

